ted_sws/data_manager/services/daily_notices_metadata_services.py

```diff
@@ -0,0 +1,156 @@
+from datetime import date, datetime, timedelta
```

Comment:
Migrate this file into supra_notice services folder and tests too

Reply:
done